build/version

```diff
@@ -5,8 +5,8 @@
 CONTAINER_BUILDVERSION_MAJOR=10
 CONTAINER_BUILDVERSION_MINOR=0
 CONTAINER_BUILDVERSION_PATCH=0
-CONTAINER_BUILDVERSION_BUILDNR=1
-CONTAINER_BUILDVERSION_DATE=20200526
+CONTAINER_BUILDVERSION_BUILDNR=4
```

Comment:
why bump from 1 to 4 ? due to 2 hotfixes ?

Reply:
> why bump from 1 to 4 ? due to 2 hotfixes ?

> That's right.